docs/docs/examples/vector_stores/BagelAutoRetriever.ipynb

```diff
@@ -1,7 +1,6 @@
 {
```

Comment:
<p>Adding "BAGEL_API_KEY" to set environment. API key can be generated from <a href="https://console.bageldb.ai/dashboard" target="_blank">https://console.bageldb.ai/</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/12594/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>